src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
I think shape and color should have string literal types. We use only 3 colors and shapes